### PR TITLE
Update site text to be relevant to new year (partial fix)

### PR DESCRIPTION
**Pull request by geoffreyyip**
Resolves items 10, 11, 12, 15, 16, 17, and 18 of #254.
Descriptions are taken directly from commits.

### Update content in driver signup form
- Changed "Phone number (cell preferred)" to "Cell phone number"
- Changed "Preferred method of contact" to "Preferred notification
  method (choose as many as you like)"
- Removed "Phone" as an option from "Preferred method of contact..."

### Update story.html and story box on front page
- Updated content. Old content was relevant to before Nov. 9, 2016
  election, not 2017.
- Configure hyperlinks to open in a new tab.
- Remove reference to 2016 in front page's story box.

### Replace 2016 countdown widget with special election info  …
The old countdown bannerbox contained an embeddable countdown widget
from 270towin.com. That widget was designed specifically for the 2016
Presidential election, and we have no way of updating the widget for the
2017  elections.

For the time being, we replace the widget with dates for the NJ and
VA state elections. A new countdown widget will need to be developed.

### 	Update nav.html twitter text
Old twitter text was designed for the 2016 US Presidential election.
This new text is more generic, and designed for 2017 elections in
general.

### 	Update accommodation section in driver sign-up form
RiderAccomodationNotes now exist as both placeholder text and as
footnotes to the <textarea> input box. Before they only existed as
placeholder text, which was difficult for vision-impaired users to see.
Furthermore, it posed a usuability problem as the helper text would
disappear as soon as the user entered any text.

This commit also adds a checkbox for "I can only communicate in a
language other than English".

### CODE REVIEW

**Comment by geoffreyyip** [comment]
I've resolved the LLC item too. Please let me know how you want to resolve the merge conflict.

**Comment by jkbits1** [comment]
@geoffreyyip I've added a note to my PR that you've already been working on this (that's why you can see a reference to that PR just above)

**Comment by richardwestenra** [comment]
Hi @geoffreyyip, thanks for this PR. Would you like me to [resolve these conflicts using the GitHub editor](https://help.github.com/articles/resolving-a-merge-conflict-on-github/), or are you happy to handle them?

File: _includes/nav.html [hunk 1]
```diff
@@ -39,7 +39,7 @@
 
         <ul class="social">
             <li role="presentation" class="social__twitter">
-                <a href="https://twitter.com/intent/tweet" class="twitter-mention-button" data-text="Support #Democracy4All this #Election2016. Sign up on @CarpoolVote: Linking drivers with voters needing a ride" data-url="http://carpoolvote.com" data-related="" data-show-count="false">Tweet to help others find out about this!</a>
+                <a href="https://twitter.com/intent/tweet" class="twitter-mention-button" data-text="Support #VotingRights in 2017 #USelections. Sign up via @carpoolvote: Linking drivers with voters needing a ride. carpoolvote.com" data-url="http://carpoolvote.com" data-related="" data-show-count="false">Tweet to help others find out about this!</a>
```
Comment (richardwestenra) [inline _includes/nav.html]: There's no need to add the URL to the tweet text, as it's already included by the `data-url` attribute. It also makes the tweet too long:
image
Please change to the following:
```
data-text="Support #VotingRights in 2017 #USelections. Sign up via @carpoolvote: Linking drivers with voters needing a ride."
```

File: pages/index.html [hunk 2]
```diff
@@ -127,10 +127,25 @@ <h2 class="bannerbox__title">I need a ride</h2>
                                 <input type="checkbox" name="TwoWayTripNeeded" id="rideReturn" /> I need a two-way trip.
                             </label>
                         </div>
+                        <div class="form-group checkbox">
+                            <label for="riderDoesNotSpeakEnglish">
+                                <input type="checkbox" name="TwoWayTripNeeded" id="riderDoesNotSpeakEnglish" /> I can only communicate in a language other than English.
```
Comment (richardwestenra) [inline pages/index.html]: `name="TwoWayTripNeeded"` is incorrect, and duplicates the previous checkbox. Please change the name attribute to a unique value. Maybe just `name="riderDoesNotSpeakEnglish"`?

File: pages/index.html [hunk 3]
```diff
@@ -127,10 +127,25 @@ <h2 class="bannerbox__title">I need a ride</h2>
...
+                            </label>
+                        </div>
                         <div class="form-group">
-                            <label for="RiderAccommodationNotes">Notes <i class="optional">Optional</i></label>
+                            <label for="RiderAccommodationNotes">Other Requirements <i class="optional">Optional</i></label>
                             <textarea class="form-input" id="RiderAccommodationNotes"
                             placeholder="Accommodation requirements e.g. service animal, assistance folding equipment, assistance entering/exiting the vehicle, child car seat or booster (please include age), must also collect non-voting assistant or companion, or anything else the driver should know." name="RiderAccommodationNotes" rows="4"></textarea>
+                            <small>
+                                Please let us know of any other accommodation requirements.
+                                <ul>
```
Comment (richardwestenra) [inline pages/index.html]: Placing `<ul>` and `<li>` elements inside a `<small>` is invalid HTML :(
I recommend replacing the `<small>` with a `<div>` (?) with either a new `small-text` class or an inline style, with `font-size: 80%`.
Does that sound like a decent solution to you?

Although maybe you could just apply the small tag to the first line, and add the class to the `<ul>`, like this: 
```
<small>Please let us know of any other accommodation requirements.</small>
<ul class="small-text">
  <li>Service animal</li>
  <li>Assistance folding equipment</li>
  <li>Assistance entering/ exiting the Vehicle</li>
  <li>Child car seat or booster (please include age)</li>
  <li>Etc.</li>
</ul>
```
```
.small-text {
   font-size: 80%;
}
```

File: pages/index.html [hunk 4]
```diff
@@ -352,11 +364,14 @@ <h2 class="bannerbox__title">How to Claim Your Vote</h2>
                 </div>
             </div>
             <div class="bannerbox" id="countdown">
-                <h2 class="bannerbox__title">2016 Election Countdown Clock</h2>
+                <h2 class="bannerbox__title">Upcoming Elections</h2>
+                <!-- Developer Note: There used to be a third-party widget that counted down to the 2016 Presidential election. A new countdown widget
+                needs to be developed. For now, we will include info on the 2017 special elections.-->
```
Comment (richardwestenra) [inline pages/index.html]: Good call 👍

File: pages/index.html [hunk 5]
```diff
@@ -352,11 +364,14 @@ <h2 class="bannerbox__title">How to Claim Your Vote</h2>
...
                 <div class="bannerbox__content">
-                    <iframe src="http://www.270towin.com/2016-countdown-clock/widget300x200.php" width="300" height="200" style="border:none; max-width:100%">
-                        Browser not supported. Visit <a href="http://www.270towin.com/">www.270towin.com</a>
-                    </iframe>
+                    <p><strong>New Jersey state primaries:</strong> June 6, 2017</p>
+                    <p><strong>Virginia state primaries:</strong> June 15, 2017</p>
+                    <p><strong>New Jersey and Virginia state elections:</strong> November 7</p>
+                    <a href="https://ballotpedia.org/State_legislative_special_elections,_2017">2017 special elections</a>
```
Comment (richardwestenra) [inline pages/index.html]: Could turn this set of `<p>` tags into an unordered list? What do you reckon?

Comment (geoffreyyip) [inline pages/index.html]: I like it. It's more semantic that way.

File: pages/index.html (hunk 3 above)
Comment (geoffreyyip) [inline pages/index.html]: How about I wrap all the text with `<small>` tags?

```
<small>Please let us know of any other accommodation requirements.</small>
<ul>
  <li><small>Service animal</small></li>
  <li><small>Assistance folding equipment</small></li>
  <li><small>Assistance entering/ exiting the Vehicle</small</li>
  ...
</ul>

CSS styling feels brittle. `font-size: 80%` may or may not correspond to `<small>` in the future.

This isn't the most elegant solution, but neither is a catch-all accommodation requirements `<textarea>`. We may want to redesign this portion soon anyways.

**Comment by geoffreyyip** [comment]
@richardwestenra I resolved the merge conflicts.

I'm ready to push changes. Lemme know if you're okay with the `<small>` tags solution I proposed.

**Comment by sasjkia** [comment]
thanks for doing this @geoffreyyip!

For the rider notes, please could you also add the bit about Spanish language (you can find the wording from the original issue). Please could you also change the "Etc" bullet to say "Anything else"?

Also, @kylejcrawford do we still need the guidance text inside the box if it is also underneath? Should we change that text?

**Comment by sasjkia** [comment]
also, do you think this text on the desktop interface would be more convenient if next to the box, rather than underneath it?

**Comment by kylejcrawford** [comment]
@sasjkia There is no need to duplicate the guidance text inside and outside the box. And because in the Location Details section earlier in the sign-up process the guidance text is within the text boxes, it'd make sense to be consistent and do that in the Vehicle Requirements section also. 

It'd be worth simplifying this copy though to something like: "Please let us know of any other accommodations you will need."

**Comment by geoffreyyip** [comment]
@sasjkia @kylejcrawford I'm moving discussion on the **Vehicle Requirements** to #300. I'm in favor of redesigning it, but this pull request isn't the place to do it.

**Comment by geoffreyyip** [comment]
@sasjkia 

>For the rider notes, please could you also add the bit about Spanish language (you can find the wording from the original issue). Please could you also change the "Etc" bullet to say "Anything else"?

I haven't been able to find anything about "Spanish" from the original issue, #254. Could you add a clarifying comment there about what you're looking to do.

Alternatively, we can also close that issue, and move remaining items to a new issue. #254's is getting too big 😛 .

**Pull request merged**